.example.env

```diff
@@ -23,12 +23,6 @@ RESOLVER=127.0.0.11
 HTTPS_HOST=${COMPUTER_HOSTNAME}
 HTTPS_PORT=443
 
-# define media and static url
-MEDIA_ROOT=/django_project/minisass/static
-STATIC_ROOT=/django_project/minisass/media
-MEDIA_URL=/media/
```

Comment:
Let's keep this in the env even if they use defaults, will help in future if they still need to be exclusively defined.